pkg/preference/preference_test.go

```diff
@@ -701,34 +701,6 @@ func TestGetPushTarget(t *testing.T) {
 	}
 }
 
-func TestFormatSupportedParameters(t *testing.T) {
```

Comment:
I remember this test throwing me off also and wanting it to remove.


But then I realized that because this test failed I discovered that I forgot to add my new option to supported parameter output. Without this test, I wouldn't know that my new option is not showing up in the help.

This could be easily avoided by storing available parameters with the description in `map[string]string` or in `list` so we can iterate over it and always be sure that all available parameters are displayed. But right now they are just independent variables.

Reply:
added it back

Reply:
Oh, I was completely wrong. This test is really useless as it will not catch if you forget to add something. It is quite opposite, if you forget to add a new variable into the output it will pass :-)

Reply:
I checked again. You are right it's pretty useless. It should just check that the formatting was correctly applied without testing against a hard coded string.